tests/test-utils.php

```diff
@@ -143,5 +143,13 @@ public function testAssocArgsToString() {
 		) ) );
 	}
 
+	public function testMaybePreviewEnv() {
+		if ( 0 === strncasecmp( 'WIN', PHP_OS, 3 ) ) {
```

Comment:
Conditions within tests are somewhat dangerous, because you can never be certain which tests are actually running.

Should we allow for an environmental override in `is_windows()`, so we can have our tests cover both scenarios by setting the environmental variable?